src/components/NewCommentForm.tsx

```diff
@@ -75,25 +161,43 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('textarea', {
+              'is-danger': isBodyError,
+            })}
+            value={body}
+            onChange={({ target }) => {
+              setBody(target.value);
+              setIsBodyError(false);
```

Comment:
maybe it's better to create a separate function to handle errors? just suggestion